docs/use_cases/model_applications/convection_allowing_models/Point2Grid_obsLSR_ObsOnly_PracticallyPerfect.py

```diff
@@ -141,19 +141,22 @@
 # --------
 #
 # .. note::
-#   `ASCII2NC <https://dtcenter.github.io/METplus/search.html?q=ASCII2NCToolUseCase&check_keywords=yes&area=default>`_
-#   `Point2Grid <https://dtcenter.github.io/METplus/search.html?q=Point2GridUseCase&check_keywords=yes&area=default>`_
-#   `RegridDataPlane <https://dtcenter.github.io/METplus/search.html?q=RegridDataPlaneToolUseCase&check_keywords=yes&area=default>`_
-#   `Python Embedding Ingest <https://dtcenter.github.io/METplus/search.html?q=PyEmbedIngestToolUseCase&check_keywords=yes&area=default>`_
-#   `Regridding in Tool <https://dtcenter.github.io/METplus/search.html?q=RegriddingInToolUseCase&check_keywords=yes&area=default>`_
-#   `NetCDF  <https://dtcenter.github.io/METplus/search.html?q=NetCDFFileUseCase&check_keywords=yes&area=default>`_
-#   `Python Embedding  <https://dtcenter.github.io/METplus/search.html?q=PythonEmbeddingFileUseCase&check_keywords=yes&area=default>`_
-#   `ConvectionAllowingModelsAppUseCase <https://dtcenter.github.io/METplus/search.html?q=ConvectionAllowingModelsAppUseCase&check_keywords=yes&area=default>`_,
-#   `NCAROrgUseCase <https://dtcenter.github.io/METplus/search.html?q=NCAROrgUseCase&check_keywords=yes&area=default>`_,
-#   `ProbabilityGenerationUseCase <https://dtcenter.github.io/METplus/search.html?q=ProbabilityGenerationUseCase&check_keywords=yes&area=default>`_,
-#   `MaskingFeatureUseCase <https://dtcenter.github.io/METplus/search.html?q=MaskingFeatureUseCase&check_keywords=yes&area=default>`_ 
-#   `HMTOrgUseCase <https://dtcenter.github.io/METplus/search.html?q=HMTOrgUseCase&check_keywords=yes&area=default>`_ 
-#   `HWTOrgUseCase <https://dtcenter.github.io/METplus/search.html?q=HWTOrgUseCase&check_keywords=yes&area=default>`_ 
 #
-
+#   * ASCII2NC
+#   * Point2Gri
```

Comment:
It looks like there is a "d" missing in "Point2Grid" can you please add that back in?

Reply:
I fixed this up and made a few other changes to this file.